packages/react/src/components/Accordion/next/Accordion.stories.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* eslint-disable no-console */
+
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import {
+  withKnobs,
+  boolean,
+  number,
+  select,
+  text,
+} from '@storybook/addon-knobs';
```

Comment:
Knobs should be removed as they don't work in the new storybook. We have to use controls instead. 

We can totally defer the addition of controls to get this PR in, but let's open an issue for it and add it to this/next sprint.

Reply:
Sounds good! Do we have a good working example using controls? I'm down to switch from them in this PR but I'm just not sure how controls work vs knobs

Reply:
@abbeyhrt Cool! Yep, there are a few. Any `*.stories.js` files with usages of "args" or "argTypes".

- `Theme.stories.js` is a pretty solid baseline example
- `Notification.stories.js` uses some additional parts of the API to apply global controls/args to all stories and additionally applies some extra ones to specific stories.